docs : Cleaned up documentation (#861)
kantord/LibreLingo: docs/CONTRIBUTING.md

# Contributing

> We welcome all contributions right from development to the course material.

Reviewer: ```suggestion
We welcome contributions to our codebase as well as the course material.
```

> In order to contribute to LibreLingo you can check our detailed [contributing guide](https://github.com/yash2189/LibreLingo/blob/fixing-doc/docs/CONTRIBUTING_GUIDE.md)

Reviewer: ```suggestion
Check out our detailed [contributing guide](https://github.com/yash2189/LibreLingo/blob/fixing-doc/docs/CONTRIBUTING_GUIDE.md)
```

* Frontend, implemented using Svelte
* Course editor, implemented using Django

The site is statically built and hosted on GitHub pages, therefore there's no real "backend" or API.
We welcome all contributions right from development to the course material.
In order to contribute to LibreLingo you can check our detailed [contributing guide](https://github.com/yash2189/LibreLingo/blob/fixing-doc/docs/CONTRIBUTING_GUIDE.md)

## Getting started

Please check out our development documentation for instructions on
[setting up the development environment.](https://librelingo.readthedocs.io/en/latest/#setting-up-the-development-environment)

> ### Pick an issue

Reviewer: Please link the `Pick an issue` section from the development docs
